frontends/web/src/routes/account/send/send.tsx

```diff
@@ -654,21 +657,11 @@ class Send extends Component<Props, State> {
                       note={note}
                       onNoteChange={this.handleNoteInput}
                     />
-                    <ColumnButtons
-                      className="m-top-default m-bottom-xlarge"
-                      inline>
-                      <Button
-                        primary
-                        onClick={this.send}
-                        disabled={this.sendDisabled() || !valid || isUpdatingProposal}>
-                        {t('send.button')}
-                      </Button>
-                      <ButtonLink
-                        secondary
-                        to={`/account/${code}`}>
-                        {t('button.back')}
-                      </ButtonLink>
-                    </ColumnButtons>
+                    <ButtonsGroup
+                      onSendButtonClick={this.send}
+                      accountCode={code}
+                      isSendButtonDisabled={sendButtonDisabled}
+                    />
                   </Column>
                 </Grid>
```

Comment:
This change hides await the 2 buttons, but does it need to be in its own re-usable component? I am a bit afraid that continuing this way the code base is getting fragmented. I slightly prefer having `<ColumnButtons>` and `<Column>` be used in the same render. Do you have strong opinion?

Reply:
I slightly prefer to keep as is and close this PR so it is easier to understand the code.

Reply:
>  Main idea here is just to slightly improve readability in send.tsx, which is as we know; very long. And breaking up into different components helps in this case. It's less about making them a reusable component.

IMO the render/JSX part is fine, the beast is in the business logic of send.tsx. Looking forward to the service improvements.